client/webserver/http.go

```diff
@@ -190,6 +196,97 @@ func (s *WebServer) handleOrders(w http.ResponseWriter, r *http.Request) {
 	})
 }
 
+// handleExportOrders is the handler for the /orders/export page request.
+func (s *WebServer) handleExportOrders(w http.ResponseWriter, r *http.Request) {
+	filter := new(core.OrderFilter)
+	err := r.ParseForm()
+	if err != nil {
+		http.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest)
+		return
+	}
+
+	filter.Hosts = r.Form["hosts"]
```

Comment:
If we need to send data in the query string, let's use `chi.URLParam` to retrieve the data as we do elsewhere. This would also enable us to reuse the encoding code from `submitFilter` in `exportOrders`. Just gotta do a `strings.Split` instead.

Reply:
Somewhere are using `chi.URLParam` because in that place there are some `must have` parameter. While in here all the parameters are optional. So I think it is harder if we use that way.

Reply:
Not sure that I follow. `chi.URLParam` returns an empty string if the parameter is not set.

Reply:
I think we might be miscommunicating here.  It does seem like path components make for a tricky interface and implementation here.  What path format are you envisioning, @buck54321?
URLParam with path parts would have the request be something like:
`/orders/export/hosts/dex.decred.org,127.0.0.1/assets/dcr,btc/statuses/`
but it feels more natural to use url query parmeters like `/orders/export/?hosts=dex.decred.org,127.0.0.1&assets=dcr,btc`
(well, I'm not sure if you'd use commas or repeat parameters actually)

Reply:
You're absolutely right. I was confusing query parameters with url parameters. I guess there is `http.Request.URL.Query()`, but your way is fine, @skynet0590. Sorry about that.